src/Lime.Messaging/Contents/Reaction.cs

```diff
@@ -0,0 +1,162 @@
+﻿using Lime.Protocol;
+using System.Runtime.Serialization;
+using static System.Net.WebRequestMethods;
+
+namespace Lime.Messaging.Contents
+{
+    [DataContract(Namespace = "http://limeprotocol.org/2014")]
+    public class Reaction : Document
+    {
+        public const string MIME_TYPE = "application/vnd.lime.reaction+json";
+        public static readonly MediaType MediaType = MediaType.Parse(MIME_TYPE);
+
+        public const string EMOJI_KEY = "emoji";
+        public const string MESSAGE_ID_KEY = "id";
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Reaction"/> class.
+        /// </summary>
+        public Reaction()
+            : base(MediaType)
+        {
+        }
+
+        /// <summary>
+        /// Gets or sets the emojis associated with the reaction.
+        /// </summary>
+        [DataMember(Name = EMOJI_KEY)]
+        public Emojis Emoji { get; set; }
+
+        /// <summary>
+        /// Gets or sets the identifier of the message to which the reaction is related.
+        /// </summary>
+        [DataMember(Name = MESSAGE_ID_KEY)]
+        public string Id { get; set; }
+
+
+        /// <summary>
+        /// Returns a <see cref="System.String" /> that represents this instance.
+        /// </summary>
+        /// <returns>
+        /// A <see cref="System.String" /> that represents this instance.
+        /// </returns>
+        public override string ToString() => Emoji.ToString();
```

Comment:
What is the use case expected for this ToString. Since we do not have the Id value here, 2 Reaction to different messages will result in the same string